web/themes/custom/pineapple/components/_card.twig

```diff
@@ -1,14 +1,27 @@
 {% set card_type = card_type|default('vertical') %}
 {% set attributes = attributes|default(create_attribute()).addClass([
   'card-container',
+  card_type,
 ]) %}
 <div {{ attributes }}>
   <div class="card-image-container">
     <img src="{{ image_url }}" alt="{{ image_alt_text }}">
+    {% if tag %}
+      <div class="highlighted-label">
+        <p>{{ tag }}</p>
+      </div>
+    {% endif %}
   </div>
   <div class="card-content u-o-3">
```

Comment:
Reduce owl class spacing value by one to `u-o-2`

Reply:
done